README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://github.com/pymc-devs/pymc3/blob/master/docs/pymc3_logo.jpg?raw=true
+.. image:: https://cloud.githubusercontent.com/assets/781659/26266864/0d462598-3c9c-11e7-8697-2e811e231694.png
```

Comment:
also, good point with the jpg logo -- I might add just the "rocket" png as well -- the letters were pixel-y on a projector, so I used the rocket and typed the remaining text to match.